src/client/renderer/r_types.h

```diff
@@ -462,7 +462,11 @@ typedef struct {
 	r_bsp_flare_t *flare;
 
 	r_pixel_t lightmap_s, lightmap_t; // lightmap texture coords
-	r_image_t *lightmap;
+
+	union {
+		r_image_t *lightmap;
+		const byte *lightmap_input; // this is only used by r_lightmap
+	};
```

Comment:
I could have done that but it seemed weird to hijack an r_image_t pointer just to store a temporary byte pointer for a single step. Do you think the cast would be less disconcerting?